### PR TITLE
Added textual data

### PR DESCRIPTION
**Pull request by madan96**
I have edited the HomePage for now. We can edit the description under Member name.

### CODE REVIEW

File: home/templates/home/index.html [hunk 1]
```diff
@@ -98,35 +98,35 @@
                 <div class="carousel-caption">
                     <div class="col-md-12 col-xs-12 text-center">
                       <h2>KOSS</h2>
-                      <h3 class="animated2"> <b>Creative</b> Modern and clean </h3>
+                      <h3> Kharagpur <b>Open Source</b> Society </h3>
                       <div class="line"></div>
-                      <p class="animated3">Unique clean design</p>
+                      <p class="animated3">CODE | DEBUG | CODE</p>
```
Comment (kshitij10496) [inline home/templates/home/index.html]: FOSS is a lot more than coding and debugging.

**CODE | TEST | DEBUG (if possible | DOCUMENT)**  looks better to me

File: home/templates/home/index.html [hunk 2]
```diff
@@ -441,7 +441,7 @@ <h4 class="feature_sub">Events we conduct </h4>
             <div class="col-md-12 col-sm-12 col-xs-12">
                 <div class="feature_header text-center">
                     <h3 class="feature_title">What <b>We do</b></h3>
-                    <h4 class="feature_sub">Lorem ipsum dolor sit amet, consectetur adipisicing elit. </h4>
+                    <h4 class="feature_sub" align="center"> We are a group of Open Source Enthusiasts who focus on something more preliminary and relevant, ”A Love for Coding”.<br>KOSS plans to hold events such as familiarizing students with terminal commands, Linux environment, Github, teach algorithms and data structures, besides conducting workshops on GUI programming, Web development , Android applications and Hackathons.</h4>
```
Comment (kshitij10496) [inline home/templates/home/index.html]: Along with GitHub, we can also mention Git (VCSs) here.

File: home/templates/home/index.html [hunk 3]
```diff
@@ -795,8 +795,8 @@ <h4 class="feature_sub">We are an amazing team of 29, hardworking and focussed !
                             <div class="mask1"></div>
                             <div class="mask2"></div>
                             <div class="info">
-                              <h3>Heading here</h3>
-                              <p>Description goes here</p>
+                              <h3>Smruti Ranjan Swain</h3>
+                              <p>Github Handle: bad-smruti</p>
```
Comment (kshitij10496) [inline home/templates/home/index.html:799]: Haha ! Ping @bad-smruti

**Comment by kshitij10496** [comment]
Good work @madan96 !
I have left my comments

File: home/templates/home/index.html (hunk 2 above)
Comment (madan96) [inline home/templates/home/index.html]: Can you please elaborate? What does VCS have to  do with gh handles?

Comment (OrkoHunter) [inline home/templates/home/index.html]: @madan96 I think he means that 'teaching GitHub' should not make it look that 'Git' has been taught or is it trivial in any sense.

Some suggestions.
`terminal commands` -> Unix tools
`algorithms and data structures` -> `Algorithms and Data Structure`
`Android application (development`

Comment (kshitij10496) [inline home/templates/home/index.html]: > What does VCS have to do with gh handles?

There is no point in talking about GitHub when the person is not familiar with Git.
Also, GitHub ain't the only Git-based platform for hosting software.
So, VCS are a more relevant topic here.

Moreover, I think `Git Development Workflow` would be a more suitable title.

File: home/urls.py~ [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
+
+from . import views
+
+app_name = 'home'
+
+urlpatterns = [
+    url(r'^$', views.index, name="index"),
+    url(r'^events$', views.events, name="events")
+]
```
Comment (evijit) [inline home/urls.py~]: @madan96 Please remove this file.

**Comment by evijit** [comment]
Thanks @madan96 👍  Merged.

**Pull request merged**